cmd/roles/roles.go

```diff
@@ -188,11 +188,7 @@ func (mr *MilvusRoles) setupLogger() {
 	roleName := paramtable.GetRole()
 	rootPath := logConfig.File.RootPath
 	if rootPath != "" {
-		if id < 0 {
-			logConfig.File.Filename = path.Join(rootPath, roleName+".log")
-		} else {
-			logConfig.File.Filename = path.Join(rootPath, roleName+"-"+strconv.FormatInt(id, 10)+".log")
-		}
+		logConfig.File.Filename = path.Join(rootPath, roleName+"-"+strconv.FormatInt(id, 10)+".log")
```

Comment:
maybe `fmt.Sprintf("%s-%d.log", roleName, id)` ?

Reply:
it seems better 🤔